pom.xml

```diff
@@ -16,6 +16,20 @@
         <selenium.version>4.0.0</selenium.version>
         <testng.version>7.4.0</testng.version>
         <webdrivermanager.version>5.0.3</webdrivermanager.version>
+        <log4j.version>2.14.1</log4j.version>
+        <aspectj.version>1.8.10</aspectj.version>
+        <slf4j-log4j12.version>1.7.32</slf4j-log4j12.version>
+        <javafaker.version>1.0.2</javafaker.version>
+        <surefire.version>2.20</surefire.version>
+        <surefire-testng.version>2.12.4 </surefire-testng.version>
+        <maven-compiler.version>3.8.1</maven-compiler.version>
+        <cart>src/test/resources/cart.xml</cart>
```

Comment:
I think it would make sense to separate version from paths with a space line, what do you think?

Reply:
done